lib/decidim/decidim_awesome/admin_engine.rb

```diff
@@ -47,6 +48,18 @@ class AdminEngine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_awesome.admin_menu" do
+        Decidim.menu :admin_user_menu do |menu|
+          if Decidim::DecidimAwesome.allow_admin_accountability
```

Comment:
```suggestion
        if Decidim::DecidimAwesome.allow_admin_accountability
          Decidim.menu :admin_user_menu do |menu|
```
No need to iterate fi no admin conf